### DISCUSSION TITLE
Expose `base` config to `Astro` global namespace

### DISCUSSION THREAD

**Discussion by lwhiteley** [withastro/roadmap#1025] [Proposal]
### Body
I saw the issue but it was closed without further discussion
https://github.com/withastro/astro/issues/3626

# Summary

currently astro exposes only `Astro.site` 
but can it be possible to also expose `base` at that same level so one could use this to create custom link components for mdx instead of copying the base path in every markdown file or astro file

# Goals

A **concise, bulleted-list** outlining the intended goals of this RFC. 

- Create a component that handles reading the base path and append to links in one place

# Example

`Astro.base`

**Comment by bluwy** [comment]
There's an existing discussion at https://github.com/withastro/roadmap/discussions/845

**Reply by lwhiteley** [reply]
Thanks I missed it
